src/main/java/com/hrznstudio/titanium/block/tile/inventory/SidedInvHandler.java

```diff
@@ -85,6 +87,22 @@ public Rectangle getRectangle() {
         return new Rectangle(this.getXPos() - renderingOffset - 3, this.getYPos() - renderingOffset - 3, 18 * this.getXSize() + renderingOffset * 2 + 3, 18 * this.getYSize() + renderingOffset * 2 + 3);
     }
 
+    public SidedInvHandler setButtonCoords(int buttonX, int buttonY) {
```

Comment:
Make this method part of the IFacingHandler interface and call it `setFacingHandlerPos`